modules/disabling-insights-operator-alerts.adoc

```diff
@@ -7,14 +7,41 @@
 [id="disabling-insights-operator-alerts_{context}"]
 = Disabling Insights Operator alerts
 
-You can stop Insights Operator from firing alerts to the cluster Prometheus instance.
+ifndef::openshift-rosa,openshift-dedicated[]
+You can stop the Insights Operator from firing alerts to the cluster Prometheus instance by editing the `support` secret. The `support` secret does not exist by default and must be created when adding custom configurations for the first time. Configurations in the `support` secret override the defaults set in the `pod.yaml` file.
+endif::openshift-rosa,openshift-dedicated[]
+ifndef::openshift-rosa,openshift-dedicated[]
+You can stop the Insights Operator from firing alerts to the cluster Prometheus instance by editing the `support` secret. The `support` secret is created by default. Configurations in the `support` secret override the defaults set in the `pod.yaml` file.
```

Comment:
```suggestion
To prevent the Insights Operator from sending alerts to the Prometheus instance of the cluster, you can edit the `support` secret. Note that this `secret` is created by default. The configurations stored in the support secret take precedence over any default settings specified in the `pod.yaml` file.
```